src/status_im/contexts/wallet/add_account/create_account/new_keypair/check_your_backup/view.cljs

```diff
@@ -65,7 +66,8 @@
         quiz-index                            (reagent/atom 0)
         incorrect-count                       (reagent/atom 0)
         show-error?                           (reagent/atom false)
-        {:keys [secret-phrase random-phrase]} (rf/sub [:wallet/create-account])]
+        {:keys [secret-phrase random-phrase]} (rf/sub [:wallet/create-account])
+        secret-phrase                         (security/safe-unmask-data secret-phrase)]
```

Comment:
hmm, not a fan of overriding vars like this. Can we avoid it? 
Is there a general consensus on doing things like this in cljs? 
@ilmotta wdyt? 🤔

Reply:
@J-Son89, @OmarBasem I did my fair share of shadowing let bindings in Clojure in the name of convenience. The consensus in the team is to not shadow, not even local bindings. AFAIR this is not in our guidelines.

Currently, clj-kondo can only protect us from shadowed vars and shadowed function args, but not shadowed local bindings.